lib/engine/step/g_1846/buy_sell_par_shares.rb

```diff
@@ -7,15 +7,31 @@ module Step
     module G1846
       class BuySellParShares < BuySellParShares
         def can_buy?(entity, bundle)
-          if @game.bundle_is_presidents_share_alone_in_pool?(bundle)
-            bundle = bundle.to_bundle
-            return false unless entity.num_shares_of(bundle.corporation) == 1
+          return unless bundle
 
-            percent = 10
-            bundle = ShareBundle.new(bundle.shares, percent)
+          bundle = bundle.to_bundle
+          corporation = bundle.corporation
+
+          if corporation.receivership? && bundle.presidents_share
+            return false if entity.num_shares_of(corporation).zero?
+
+            bundle = ShareBundle.new(bundle.shares, 10)
+          end
+
+          super(entity, bundle)
```

Comment:
hmm interesting since you mutate bundle here, 

i'm not sure, but if you just call super, (without the args) will it do the same thing?

Reply:
In stock Ruby, super with no arguments works great; however, I confirmed I need to explicitly pass the parameters here. Perhaps a limitation/bug with Opal?